pyproject.toml

```diff
@@ -41,7 +41,7 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "itk~=5.4.0",
+    "itk~=5.4.2",
```

Comment:
SuperElastix/elastix still supports ITK 5.4.1, so I guess it might as well say `"itk~=5.4.1"`, right?

Reply:
There are no python packages generated for 5.4.1.

Reply:
Thanks @dzenanz But if I understand correctly, the expression `"itk~=5.4.1"` would match both `5.4.1` _and_ `5.4.2`, even if `5.4.1` doesn't exist  😸

Reply:
Since we are building against `itk 5.4.2`, Bumping the compatible release specification is a good  idea:

  https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release

There are not necessarily binary compatibility issues, but this is a good practice.